### DISCUSSION TITLE
Are MD parameters from xtb passed on to QCxMS?

### DISCUSSION THREAD

**Discussion by tobigithub** [qcxms/QCxMS#51] [Q&A] [answered]
Hi,
I was wondering if the MD parameters for XTB are passed on to QCxMS?
Do  they have any influence when we add them to the qcxms.in file? That includes:
`````
mdtemp 298.15 # MD thermostat/initial siman/GBSA temperature
mdtime 50.0 # MD run time in ps
mddump 1000. # dump (=optimize) structure in siman every mddump fs
mddumpxyz 50. # MD dump step in fs for xyz output file, MUST BE .eq. mdstep for power IR spectra
velodump 0 # set to 1 if dumps (trj file) should contain velocities
nvt 1 # use thermostat (=1, =0 for NVE)
mdskip 500 # skip interval in -mdav, -mdopt
mdstep 4.0 # MD time step in fs (automatically determined if < 0), could be 4-5 fs with shake =2, md_hmass=4
md_hmass 4 # increase hydrogen mass to this value in amu (at const. tot. mass) allowing large time steps (=0 off)
shake 2 # shake on (=0: off which is default) for XH bonds only (=1), or all bonds (=2)
nsiman 3 # number of siman annealing blocks
ewin_conf 20.0 # energy window (kcal) for considering conformers
tend 1000 # highest siman annealing temperature (very system specific)
enan 0 # include enantiomers in siman (=1)
````
Source: http://bbs.keinsci.com/thread-10056-1-1.html

Thank you!
T.

**Comment by JayTheDog** [comment, marked as answer]
Hi, 
no, the QCxMS Input is handled completely different from the tblite library. GFN-xTB is only used to do single points with the given settings. You can, however, influence the MD by different parametes which can be found in the [qcxms docs](https://xtb-docs.readthedocs.io/en/latest/qcxms_doc/qcxms_run.html#input-keywords-in-qcxms-in). I've just added some more keywords. 
However, some of the settings you request do not make sense in the aspect of QCxMS (nvt, shake, ewin_conf), so it might not be of many use to implement this as a feature.

**Comment by tobigithub** [comment]
@JayTheDog 
Thanks!
Tobias
